frontend/src/components/messages/NewRequestDialog.vue

```diff
@@ -395,15 +396,7 @@ export default {
     this.PHONE_FORMAT = PHONE_FORMAT
     this.EMAIL_FORMAT = EMAIL_FORMAT
     this.AUTO_REPLY_MESSAGE = 'Your change request is complete.'
-    this.requestSubCategorySortOrder = [
-      "Organization Details",
-      "Facility Details",
-      "Organization phone/email",
-      "Facility phone/email",
-      "Add/change a licence",
-      "Other"
-    ]
-    this.sortedRequestSubCategories = this.sortRequestSubCategories(this.requestSubCategories, this.requestSubCategorySortOrder)
+    this.sortedRequestSubCategories = this.requestSubCategories.sort((a, b) => a.displayOrder - b.displayOrder)
```

Comment:
instead of sort subcategories in frontend, can we add this one to backend? also please use const instead of let.
const response = await getOperation('ofm_subcategories?$orderby=ofm_display_order')

Reply:
Fixed.